capstone-backend/src/main/java/com/google/sps/data/Book.java

```diff
@@ -1,35 +1,27 @@
 package com.google.sps.data;
 
 
-public class Book {
+public class Book extends BaseEntity {
     
-  private final String id;
   private final String userID;
   private final String gbookID;
-  private boolean hasRead;
   private int rating;
   private String review;
 
  
-  public Book(String id, String userID, String gbookID, boolean hasRead, int rating, String review) {
-    this.id = id;
+  public Book(String userID, String gbookID, int rating, String review) {
     this.userID = userID;
     this.gbookID = gbookID;
-    this.hasRead = hasRead;
     this.rating = rating;
     this.review = review;
   }
  
-  public Book(String id, String userID, String gbookID) {
-    this(id, userID, gbookID, false, -1, "");
+  public Book(String userID, String gbookID) {
+    this(userID, gbookID, -1, "");
```

Comment:
Even though we're not currently using the rating or review, it's a good idea to continue to include this here for where we are able to implement it in the future!

Reply:
Thanks!